content/post/adding_context_with_annotations.md

```diff
@@ -0,0 +1,106 @@
+
+<!--more-->
+
+#### Why taking a mental note isn’t a scalable solution:
+* A standard practice among analytics team is to rely on historical data to measure the success of a recent campaign or product launch. If you fail to account for a major event in your ‘pre’ period, it could mean an analysis and recommendation that are not sound.
+* At one point or another, a company executive will most certainly probe you to diagnose an anomaly or shift in performance (read: a dip in performance) -- if you can’t recall all of the events that occurred around the start of the shift there is a good chance that you’ll find yourself going down a rabbit hole try to figure it out.
+* As your company grows, there will undoubtedly be new members joining the analytics team. New analysts can’t possibly be expected to remember major events that they weren’t there to experience. Similarly, veteran employees who were there to experience major events leave the organization at one point or another.
+
+In this post I will walk through the steps I take to integrate annotations into my data models and visualizations. I will refer to an imaginary home goods e-commerce company called “Utensell” to guide the walkthrough.
+
+### Step 1: Whenever something notable happens, record it in a google spreadsheet. Easy!
+
+[screenshot_1]: https://github.com/locallyoptimistic/LocallyOptimistic/blob/master/static/img/annotations-google-sheet.jpg
+
+Examples of things I keep note of:
+* Product Launches (e.g., launched new silverware sku)
+* Marketing Promotions - both online and offline (e.g., pulled back on marketing spend temporarily)
+* Feature Releases (e.g., Utensell introduced filtering functionality to our products page)
+* Press (e.g., Utensell founders featured on the How I Built This podcast)
+* Onsite Bugs (e.g., new users were not able to sign up for the Utensell newsletter)
+* Analytics Bugs (e.g., accidentally stopped capturing clicks on the “remove item” button in checkout)
+* Competitors Behavior (e.g., Amazon prime day)
+* Fraud (e.g., surge in bot traffic)
+
+Depending on the business, it may make sense to keep note of some additional metadata. In this example, I have a boolean flag to indicate whether there was a discount promotion associated with the event. I also keep track of links that are relevant to the notable event (e.g., links to press articles) in a separate column.
+
+_Daily Time Spent: I generally spend 1-5 minutes per day on this step._
+
+
+### Step 2: Automate the formatting and uploading of your annotations log to your database.
+
+I use the [datasheets](https://github.com/Squarespace/datasheets) python library (created by the team at Squarespace) to reformat and upload my annotations log table into Utensell’s data warehouse. But you can use any method you prefer! [Quick shoutout to Rujuta Kortikar, who built V1 of this script using the gspread library before datasheets was released].
```

Comment:
These square brackets are doing weird things to the markdown. I don't think this needs to be a parenthetical statement, so I'd just drop the brackets.